src/api/app/controllers/webui/users_controller.rb

```diff
@@ -19,6 +19,7 @@ def index
   def show
     @groups = @displayed_user.groups
     @involved_items_service = UserService::Involved.new(user: @displayed_user, filters: extract_filter_params, page: params[:page])
+    @comments = paged_comments
```

Comment:
@hellcp-work maybe we can already check the policy here, and only fetch the comments if we in the end really going to render them. Might save a query here and there.